jupyter_book/cli/main.py

```diff
@@ -139,6 +139,11 @@ def main():
     default=False,
     help="[pdflatex] Enable build of PDF files for each individual page",
 )
+@click.option(
+    "--jobs",
+    default=1,
```

Comment:
maybe this could default to https://docs.python.org/3/library/multiprocessing.html#multiprocessing.cpu_count ?

Reply:
@proppy, I'd be happy to work on this due to inactivity.